src/logic/operate.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable import/extensions */
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  let res = '';
+  switch (operation) {
+    case '+':
+      res = Big(numberOne).plus(numberTwo).toString();
+      break;
+    case '-':
+      res = Big(numberOne).minus(numberTwo).toString();
+      break;
+    case 'X':
+      res = Big(numberOne).times(numberTwo).toString();
+      break;
+    case '/':
+      res = numberTwo === '0' ? res = 'Infinity' : Big(numberOne).div(numberTwo).toString();
```

Comment:
[Optional]

Wouldn't be the same if it's like this :point_down: ? :thinking: 

```suggestion
      res = numberTwo === '0' ? 'Infinity' : Big(numberOne).div(numberTwo).toString();
```

Reply:
Yeap, I did not notice that. Thanks a lot, for mentioning that.